### ISSUE TITLE
Museum display user accessibility

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#49]
The museum display needs to feel easy to use for an unexperienced user....

1.) The current UI relies on mouseover to show what all the small icons do.  All icons should be large and include as part of them enough text for example it might be that the buttons are larger and have text on.

2.) It needs to be obvious to the user from the start what the display is showing (tree of life etc etc.) and what they can do with it (search, common ancestor, back to start, etc etc)

**Comment by jrosindell** [comment]
3.) Simplify the options - remove zoom in, zoom out, user settings, and information submenu.  So, key features are a) search, b) popular species, c) common ancestor, d) reset view, e) move up to ancestor, f) your location (compass), g) information (which just opens one page and has no sub menu), h) and intelligent tutorial (which might involve a help button).  most of these features already exist in the UI and just need to be adapted.

**Comment by jrosindell** [comment]
4.) We think the UI development should be done in two distinct coding sprints with a light touch - i.e. making use of what we already built wherever possible. In between these two sprints we need to hit the pause button and arrange some user testing to be done.

**Comment by lentinj** [comment]
As I've commented in #18, I think the zoom in / out buttons should stay. Instead, the "back to start" button should go. The tree root is too abstract for an average user like meself to start at, and there's a lot of zooming until I get to something familiar. OTOH, going backwards with previous ancestor is quite instructive. Or am I missing a use-case?

But regardless, doing a prototype UI first makes a lot of sense.

**Comment by jrosindell** [comment]
I like what you're suggesting, however I've noticed a type of user that wants first of all to zoom out and see the whole before they are willing to engage properly with exploring the interesting areas.  Maybe we could satisfy such a user by making the popular places include the root of the tree and also include flight animations (rather than jumps) so they can quickly see various areas in the context of each other, just by pressing the popular areas buttons.

**Comment by lentinj** [comment]
> Maybe we could satisfy such a user by making the popular places include the root of the tree

Yup, that works.

> also include flight animations (rather than jumps) so they can quickly see various areas in the context of each other, just by pressing the popular areas buttons.

FWIW I've gotten very used to OTOP doing that, and miss it when I go back to another view.

**Comment by jrosindell** [comment]
> FWIW I've gotten very used to OTOP doing that, and miss it when I go back to another view.

I completely agree, user feedback on the main site also thinks so too, so I'd propose making the whole site do the smooth flights all the time and have an option in settings to turn it off
